### ISSUE TITLE
[Bug] Selecting date range shows no data

### ISSUE THREAD

**Issue by rcontiuk** [piwik/piwik#22867]
### What happened?

When selecting date range, the reports (Visit Over Time, Pages report, Channel Type) show no data. As opposed to selecting the Month, all reports are populated correctly in Matomo.

Selecting the Visits Over Time report, it shows no data:

![image](https://github.com/user-attachments/assets/7705291c-0f9b-4522-84ca-11dcdef48602)

### What should happen?

The Visits Over Time report should show data as highlighted in red below:

![image](https://github.com/user-attachments/assets/0fa4b05a-46bd-4fc8-b9cb-e3bbc93a708e)

### How can this be reproduced?

Yes, both users and Matomo can reproduce the issue.

The raw data clearly exists but not shown with custom range.

### Matomo version

Cloud

### Validations

- [X] Read our [Contributing Guidelines](https://github.com/matomo-org/matomo/blob/5.x-dev/CONTRIBUTING.md).
- [X] Follow our [Security Policy](https://github.com/matomo-org/matomo/blob/5.x-dev/SECURITY.md).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating duplicates.
- [X] The provided steps to reproduce is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the Bug.

**Comment by sos-fbinder** [comment]
I've got the same problem in version 5.2.0 on my self-hosted Matomo.

**Comment by sgiehl** [comment]
@sos-fbinder Are there any errors logged for you in the log file?

**Comment by sos-fbinder** [comment]
> @sos-fbinder Are there any errors logged for you in the log file?

No, I can't spot any unusual entries in my matomo logfile. No errors at all.

![Bildschirmfoto_2024-12-19_um_10 13 01](https://github.com/user-attachments/assets/4c2cd588-76eb-4a35-ab08-45703a47a8c4)

At the screenshot you can see (in the graph) the time I installed the update. But there are Visits (Besuche) in the last 24 hours tracked by Matomo (Letzte 24 Stunden), but not shown in the graph.

**Comment by sos-fbinder** [comment]
It seems that my archive cronjob was the problem. It stopped working with the update. The archive log told me, that the archiving could not run because there where already three running instances. I could not figure out what is running, because the cronjob only run once and with `ps ex` I do not see running instances...

So I reconfigured my cronjobs and split them to archive site by site one after the other and with the flag `--concurrent-archivers=-1`. And now I see data in the graph images.

**Comment by Chardonneaur** [comment]
I got someone who got this issue today.

**Comment by sos-fbinder** [comment]
I think this is related to this issue https://github.com/matomo-org/matomo/issues/22925. See my answer above.
